examples/experimental/dagster-airlift/dagster_airlift/core/airflow_defs_data.py

```diff
@@ -1,14 +1,27 @@
-from typing import AbstractSet, Optional
+from functools import cached_property
+from typing import AbstractSet, Optional, cast
 
-from dagster import AssetKey
+from dagster import AssetKey, Definitions
 from dagster._record import record
+from dagster._serdes.serdes import deserialize_value
 
 from dagster_airlift.core.serialization.serialized_data import SerializedAirflowDefinitionsData
+from dagster_airlift.core.utils import get_metadata_key
 
 
 @record
 class AirflowDefinitionsData:
-    serialized_data: SerializedAirflowDefinitionsData
+    instance_name: str
+    resolved_airflow_defs: Definitions
```

Comment:
Something about this name is a bit off. I think "resolved" is an implementation detail that will be confusing to the user. 

The word "mapping" will appear in metadata. E.g. "task mapping" Maybe let's standardize on that terminonology,

These are dagster definitions that are mapped from entities in the AirflowInstance.

```python
class AirflowInstanceDefinitionsData:
    instance_name: str
    mapped_defs: Definitions
```

?

Reply:
I buy that. mapped_defs works for me.

Reply:
Feel free to change at the tip or your stack to avoid rebase hell

Reply:
🙏 thx for your understanding